lib/src/DOtherSide.cpp

```diff
@@ -97,7 +95,7 @@ void register_meta_types()
 }
 
 // jrainville: I'm not sure where to put this, but it works like so
-QTranslator *m_translator = new QTranslator();
+static QTranslator *m_translator = new QTranslator();
```

Comment:
Unnamed namespace is the preferred way for static linking in modern c++ according to core guidelines. Probably new is not needed as well. And `m_` suggests member, `g_` prefix would be more appropriate here.